flink-yarn/src/test/resources/log4j-test.properties

```diff
@@ -18,7 +18,7 @@
 
 # Set root logger level to OFF to not flood build logs
 # set manually to INFO for debugging purposes
-log4j.rootLogger=OFF, testlogger
+log4j.rootLogger=INFO, testlogger
```

Comment:
I think we always leave it off in the master.

Reply:
yes, my bad. Will correct it.